src/condor_utils/secure_file.cpp

```diff
@@ -287,14 +287,17 @@ read_secure_file(const char *fname, void **buf, size_t *len, bool as_root, int v
 	}
 
 	// now read the entire file.
+	// Some callers assume the returned buffer is NUL-terminated, so
+	// allocate an extra byte for a NUL.
```

Comment:
That would be a bug in the caller, not appropriate to fix that here.

Reply:
Fair point. I wanted to have something ready quickly before disappearing on vacation. But the code freeze has been delayed, so I'll do a better fix in the caller.